nntrainer/src/util_func.cpp

```diff
@@ -148,4 +148,47 @@ float reluPrime(float x) {
     return 1.0;
   }
 }
+
+// This is 2D zero pad
+// TODO : Optimize for multi dimention padding
+Tensor zero_pad(int batch, Tensor const &in, unsigned int const *padding) {
```

Comment:
Small thought: this function should prepare to be compatible with 3D zero pad or change the name to `zero_pad_2d` because now it is in `utill_func.cpp` and `uint * padding`  cannot determine if current input is 2D or 3D.

Please ignore above, I think it is good for now :)